radixdlt-java/src/main/java/com/radixdlt/client/core/network/jsonrpc/RadixJsonRpcClient.java

```diff
@@ -239,6 +219,54 @@ protected boolean tryClose() {
 		return this.jsonRpcSubscribe(method, new JsonObject(), notificationMethod);
 	}
 
+	public Observable<JsonObject> observeNotifications(String notificationMethod, String subscriberId) {
```

Comment:
`subscriberId` is the first argument for all the other methods, so it should be first here as well I think. Consistency :)

Reply:
Mmm this is actually a lower level method which exposes a way to specify the notification method string whereas the others specify it for the caller. It should be `notificationMethod` first as it is higher in the hierarchy within the json rpc method object.

Reply:
Makes sense